bin/opm

```diff
@@ -2195,6 +2198,57 @@ sub rebase_path ($$$) {
     return undef;
 }
 
+sub do_clean {
+    my @valid_options = ('dist');
+    
+    if (@_ == 0) {
+        err "no clean option specified.\n";
```

Comment:
I think "dist" is a clean *argument* instead of an option. In the CLI terminology, options are usually something like `--cwd` and `--help`.